src/app/features/change-history/change-log/change-log.component.scss

```
@@ -0,0 +1,71 @@
+}
+
+/* enough room for a few field chips side by side; without it every chip wraps
+   onto its own line and the rows grow very tall */
+.changed-fields-cell {
+  min-width: 240px;
+}
+
+/* match the action badge next to them, which sets its own 12px; the chip
+   default is larger and made the field names shout over the action */
+.changed-fields-cell
+  ::ng-deep
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)